### ISSUE TITLE
Connection to homeduino is opened AFTER the devices are declared. => unnecessary errorss

### ISSUE THREAD

**Issue by DavidZH** [pimatic/pimatic#534]
Since the last few updates during startup I get errors:
```error: Could not get attribute value of Temp woonkamer.temperature: First call connect!```
In the log I see that the homeduino connection is made AFTER this error.

I can not recall having these errors in 0.8.37.

You can ignore and dismiss them because within a few seconds these values are received anyway.
But still I think it's unnecessary.

**Comment by DDNRN** [comment]
Same for me since last one or two updates.
````
...
10:30:53info [pimatic-homeduino]: Connected to homeduino device.
10:30:52warn [pimatic]: Could not update variable DHT22.humidity: First call connect!
10:30:52warn [pimatic]: Could not update variable DHT22.temperature: First call connect!
10:30:52error [pimatic]: Could not get attribute value of Feuchtesensor.humidity: First call connect!
10:30:48error [pimatic]: Could not get attribute value of Feuchtesensor.temperature: First call connect! 
...

**Comment by sweetpi** [comment]
Thanks for reporting, should be fixed in pimatic-homeduino v0.8.40. Plese reopen if the problem still exists.

**Comment by DavidZH** [comment]
Great! I will test that tomorrow.
I did forget to mention that the device that gave the error was DHT sensor directly on the Homeduino board. The HomeduinoRF sensors did not produce warning or errors.
